WORKSPACE

```diff
@@ -47,13 +103,44 @@ http_archive(
 
 # Proto rules
 
+http_archive(
+    name = "zlib",
+    # patch_args = ["-p1"],
+    patches = [
+        # from https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/zlib/1.3.1.bcr.3/patches/add_build_file.patch
+        "//buildpatches:zlib.patch",
+    ],
+    sha256 = "9a93b2b7dfdac77ceba5a558a580e74667dd6fede4585b91eefb60f03b72df23",
+    strip_prefix = "zlib-1.3.1",
+    urls = ["https://github.com/madler/zlib/releases/download/v1.3.1/zlib-1.3.1.tar.gz"],
+)
+
+http_archive(
+    name = "com_google_protobuf",
+    integrity = "sha256-YMGrS+/p0Kl1wjRLVRG/a0T5HsPhQmyHj1a/MKBYnEM=",
+    strip_prefix = "protobuf-29.2",
+    urls = ["https://github.com/protocolbuffers/protobuf/releases/download/v29.2/protobuf-29.2.zip"],
```

Comment:
Nit: Let's not downgrade this. Use the current version or upgrade to the latest.